packages/common/src/tsconfig/TsConfigResolver.ts

```diff
@@ -44,8 +44,8 @@ export class TsConfigResolver {
         for (let fileName of configFileContent.fileNames) {
             const filePath = fileSystem.getStandardizedAbsolutePath(fileName);
             const parentDirPath = FileUtils.getDirPath(filePath);
+            files.add(filePath);
             if (dirInProject(parentDirPath) && fileSystem.fileExistsSync(filePath)) {
-                files.add(filePath);
                 directories.add(parentDirPath);
```

Comment:
What are `directories` used for? No usage is found in this project.

Reply:
It's used in the ts-morph project (not `@ts-morph/bootstrap`).

Reply:
I think maybe this should always add the directory as well, but I will look into this after merge.